### PR TITLE
[v7r0] A few fixes for running jobs locally

### PR DESCRIPTION
**Pull request by fstagni**
BEGINRELEASENOTES

*API
CHANGE: run jobs locally: use $DIRAC for $DIRACROOT

ENDRELEASENOTES

### CODE REVIEW

File: Interfaces/API/Dirac.py [hunk 1]
```diff
@@ -860,19 +863,23 @@ def runLocal(self, job):
...
+    # We don't use DIRAC.rootPath as we assume that a DIRAC installation is already done at this point
+    os.environ['DIRACROOT'] = os.environ['DIRAC']
+    self.log.verbose('DIRACROOT = %s' % (os.environ['DIRACROOT']))
 
     if 'Executable' in parameters:
       executable = os.path.expandvars(parameters['Executable'])
     else:
       return self._errorReport('Missing job "Executable"')
 
+    if '-o LogLevel' in arguments:
```
Comment (chaen) [inline Interfaces/API/Dirac.py]: Do I understand correctly that you are forcing the logLevel to debug ? Is that really ok ? 
Also, there could be several `-o` options no ?

Comment (fstagni) [reply Interfaces/API/Dirac.py]: Yes, but this is purely for running jobs "locally", meaning for test (BTW, for some historical reasons these test functions are in the API, and that is wrong of course). 
I can of course code a better solution than the one above (which was a quick and dirty one for running my own tests...).

File: Interfaces/API/Dirac.py [hunk 2]
```diff
@@ -735,13 +736,14 @@ def runLocal(self, job):
...
     parameters = res['Value']
+    self.log.debug("Extracted job parameters from JDL", parameters)
 
     arguments = parameters.get('Arguments', '')
 
     # Replace argument placeholders for parametric jobs
     # if we have Parameters then we have a parametric job
     if 'Parameters' in parameters:
-      for par, value in parameters.iteritems():
+      for par, value in parameters.items():
```
Comment (atsareg) [inline Interfaces/API/Dirac.py]: Did not we agree that if we do not want iterator we should use list(xxx.iteritems()) ?

Comment (atsareg) [reply Interfaces/API/Dirac.py]: https://github.com/DIRACGrid/DIRAC/pull/4243#issuecomment-523920369

**Pull request merged**